cloudant-sync-datastore-core/src/test/java/com/cloudant/sync/replication/CouchClientWrapperTest.java

```diff
@@ -57,8 +57,9 @@
 
     @Before
     public void setup() throws IOException {
-        remoteDb = new CouchClientWrapper(super.getCouchConfig(CLOUDANT_TEST_DB_NAME + System
-                .currentTimeMillis()));
+        CouchConfig config = super.getCouchConfig(CLOUDANT_TEST_DB_NAME + System
+                .currentTimeMillis());
+        remoteDb = new CouchClientWrapper(config.getRootUri(), config.getRequestInterceptors(), config.getResponseInterceptors());
```

Comment:
Looks like there are a few "long" statements in this file.  They are lines 62, 83, 190, 232, and 310.

I know it is sort of nitpicky but it definitely would help when it comes to reviewing for the code to be split across multiple lines.